src/static/riot/competitions/form/_leaderboard.tag

```diff
@@ -330,17 +334,23 @@
 
         self.one("mount", function () {
             //$(".tooltip").popup()
-            $(".dropdown").dropdown()
+            $(".dropdown", self.root).dropdown()
 
 
             // *NOTE* Assigning columns this way gets it out of "opts" and makes it namespace properly!
             self.columns = self.opts.columns
+            self.primary_index = self.opts.primary_index
+            self.selected_leaderboard = self.opts.leaderboard_index
         })
 
         self.on("update", function () {
             // Force refresh of dropdown on update, otherwise they don't show new elements and
             // don't reflect new names
-            //$(".dropdown").dropdown("refresh")
+            let $dropdown = $(".dropdown", self.root)
```

Comment:
why dollar sign here?

Reply:
cause it's the naming convention I learned for assigning a jQuery object to a variable.

Reply:
Let's just do normal variable names not PHP style or something :)